Classes/Emogrifier.php

```diff
@@ -445,19 +428,21 @@ protected function process(\DOMDocument $xmlDocument)
      */
     private function mapAllInlineStylesToHtmlAttributes(\DOMXPath $xPath)
     {
-        $nodesWithStyleAttributes = $xPath->query('//*[@style]');
-        if ($nodesWithStyleAttributes === false) {
+        try {
+            $nodesWithStyleAttribute = $this->getAllNodesWithStyleAttribute($xPath);
+        } catch (\RuntimeException $e) {
             if ($this->debug) {
                 throw new \RuntimeException(
-                    'Possibly malformed DOMXPath query expression or invalid DOMXPath object.',
-                    1508437884
+                    $e->getMessage(),
+                    1508437884,
+                    $e
```

Comment:
Nitpick: I think this will fit in one line again. Could you please reformat this?